pyproject.toml

```diff
@@ -79,16 +82,69 @@ addopts = [
 
 
 [tool.autopep8]
-max_line_length = 180
+max_line_length = 150
 experimental = true
 
 
 [tool.mypy]
 ignore_missing_imports = true
-follow_imports = "silent"
 show_column_numbers = true
 strict = true
+exclude = [
+    "src/dsp_tools/models",           # TODO: activate this
+    "src/dsp_tools/import_scripts",   # TODO: activate this
+    "test/unittests",                 # TODO: activate this
+    "test/e2e",                       # TODO: activate this
+]
+
 
 [tool.isort]
 multi_line_output = 3
 known_first_party = ["dsp_tools"]
+
+
+[tool.pylint.MASTER]
+ignore-paths = [
+    "src/dsp_tools/models/.*$",           # TODO: activate this
+    "src/dsp_tools/import_scripts/.*$",   # TODO: activate this
+    "test/unittests/.*$",                 # TODO: activate this
+    "test/e2e/.*$"                        # TODO: activate this
```

Comment:
These are ignored for the moment. The long term goal is to activate them one day.